lib/rules/no-extra-semicolons/index.js

```diff
@@ -53,6 +53,10 @@ const rule = function (actual) {
     }
 
     root.walk(node => {
+      if (node.mixin) {
```

Comment:
I'm not 100% satisfied with this current solution here and below, but we can't use `isStandardSyntaxRule` here since it checks for `--custom-property-set: {}`, thus failing other tests which uses that.

While working on this, though, I found myself thinking of a `isLessRule` util. Would that be of interest or is it too specific since it's a non-standard syntax thing? I was thinking that the `isLessRule` util would consist of the Less-specific checks from `isStandardSyntaxRule` and that `isStandardSyntaxRule` could make use of `isLessRule` instead. 

Using a `isLessRule` here in `no-extra-semicolons` would also catch Less `&:extend(.foo)` rules which aren't currently ignored (`&:extend(.foo)` can of course be caught and ignored without it too, but a util would be a better fit IMO).